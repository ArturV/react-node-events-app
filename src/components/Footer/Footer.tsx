import Box from "@mui/material/Box";
import Container from "@mui/material/Container";
import Typography from "@mui/material/Typography";

export const Footer = () => {
  return (
    <footer>
      <Box component="footer" sx={{ bgcolor: "background.paper", py: 6 }}>
        <Container maxWidth="lg">
          <Typography
            variant="subtitle1"
            align="center"
            color="text.secondary"
            component="p"
          >
            Copyright 2023
          </Typography>
        </Container>
      </Box>
    </footer>
  );
};
