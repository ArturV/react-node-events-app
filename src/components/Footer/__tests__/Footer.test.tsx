import { render, screen } from "@testing-library/react";
import { Footer } from "../Footer";
import renderer from "react-test-renderer";

describe("Footer", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(<Footer />);

    // expect(tree).toMatchSnapshot();
  });

  it("should render copyright", () => {
    render(<Footer />);

    const copyright = screen.getByLabelText("copyright");

    expect(copyright).toBeVisible();
    expect(copyright).toBeInTheDocument();
  });
});
