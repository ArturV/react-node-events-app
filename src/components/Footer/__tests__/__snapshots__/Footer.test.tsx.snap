// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<footer>
  <footer
    className="MuiBox-root css-1k76al1"
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <p
        aria-label="copyright"
        className="MuiTypography-root MuiTypography-subtitle1 MuiTypography-alignCenter css-1xatcrf-MuiTypography-root"
      >
        Copyright 2023
      </p>
    </div>
  </footer>
</footer>
`;
